packages/components/CHANGELOG.md

```diff
@@ -11,6 +11,27 @@
 -   `ToolsPanel`: atomic one-step state update when (un)registering panels ([#65564](https://github.com/WordPress/gutenberg/pull/65564)).
 -   `Navigator`: fix `isInitial` logic ([#65527](https://github.com/WordPress/gutenberg/pull/65527)).
 -   `ToggleGroupControl`: Fix arrow key navigation in RTL ([#65735](https://github.com/WordPress/gutenberg/pull/65735)).
+-   `ToggleGroupControl`: indicator doesn't jump around when the layout around it changes ([#65175](https://github.com/WordPress/gutenberg/pull/65175)).
+-   `Composite`: fix legacy support for the store prop ([#65821](https://github.com/WordPress/gutenberg/pull/65821)).
```

Comment:
Not sure how we're supposed to deal with the changelog 😅 I assume we should just add the entry for the changes we're cherry-picking, but would that just cause more merge conflict hassles down the line?

Reply:
> would that just cause more merge conflict hassles down the line?

I don't know, but it makes sense that this PR only adds the relevant CHANGELOG entry